x-pack/plugins/stack_alerts/public/rule_types/es_query/test_query_row/test_query_row_table.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { css } from '@emotion/react';
+import {
+  EuiDataGrid,
+  EuiPanel,
+  EuiDataGridColumn,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiBadge,
+  EuiSpacer,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+const styles = {
+  grid: css`
+    .euiDataGridHeaderCell {
+      background: none;
+    }
+    .euiDataGridHeader .euiDataGridHeaderCell {
+      border-top: none;
+    }
+  `,
+};
+
+export interface TestQueryRowTableProps {
+  rawResults: { cols: EuiDataGridColumn[]; rows: Array<Record<string, string | null>> };
+  alerts: string[] | null;
+}
+
+export const TestQueryRowTable: React.FC<TestQueryRowTableProps> = ({ rawResults, alerts }) => {
+  return (
+    <EuiPanel style={{ overflow: 'hidden' }} hasShadow={false} hasBorder={true}>
+      <EuiDataGrid
+        css={styles.grid}
+        aria-label="Test query grid"
```

Comment:
**question:** should this be translated?

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/164073/commits/cf0b33fcbd05d60910a01acbe96332f8dec5b0d1